pages/windows/replace.md

```diff
@@ -24,7 +24,7 @@
 
 `replace {{path/to/file_or_directory}} {{path/to/destination}} /w`
 
-- Replace all files in subfolders of the destination:
+- Replace all files in subdirectories of the destination:
```

Comment:
We use just `replace: ,,,` as prefix for the commit, without include the platform specific directory.

Reply:
Okay, thank you for the heads up. Rename the commit & force push or leave it as is?